recipe/conda_build_config.yaml

```diff
@@ -593,8 +591,8 @@ tk:
   - 8.6                # [not ppc64le]
 tiledb:
   - 1.7.0
-vc:                    # [win]
-  - 14                 # [win]
+vc:                      # [win]
+  - 14.1                 # [win]
```

Comment:
@isuruf should we keep this at 14?

Reply:
14.1 doesn’t match the pinning and conda will pickup the later version anyways.

Reply:
Yes, let's keep it at 14

Reply:
done